cmd/influx/cli/cli.go

```diff
@@ -255,14 +262,17 @@ func (c *CommandLine) Connect(cmd string) error {
 		return fmt.Errorf("Could not create client %s", err)
 	}
 	c.Client = cl
-	if _, v, e := c.Client.Ping(); e != nil {
+	if _, _, e := c.Client.Ping(); e != nil {
 		return fmt.Errorf("Failed to connect to %s\n", c.Client.Addr())
-	} else {
-		c.ServerVersion = v
```

Comment:
__golint__ ask to move it out of the else block. But I removed by accident. Fixed now.